### PR TITLE
modify error msg when login by 'curl' failed

### PR DESCRIPTION
**Pull request by xuweibj**
#4207

```
# rpower cn01 stat
cn01: BMC did not respond within 10 seconds, retry the command
```

### CODE REVIEW

**Comment by whowutwut** [comment]
Also partially resolves my comment in #4264  to not use 503 on our own error

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 1]
```diff
@@ -586,7 +586,7 @@ sub process_request {
 
     foreach my $node (keys %node_info) {
         if (!$valid_nodes{$node}) {
-            xCAT::SvrUtils::sendmsg([1, $::RESPONSE_SERVICE_UNAVAILABLE], $callback, $node);
+            xCAT::SvrUtils::sendmsg([1, "Unavailable now, please try again later"], $callback, $node);
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: Suggest: 

`BMC did not respond within $timeout seconds, retry the command.`

Feel free to amend the commit, it's 1 line...

**Comment by whowutwut** [comment]
@xuweibj I am ok with the changes, prefer to have that message changed first and then feel free to merge overnight

**Comment by zet809** [comment]
Thx, @whowutwut , let's merge it.

**Pull request merged**